packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/__tests__/quanticGeneratedAnswer.test.js

```diff
@@ -72,19 +143,125 @@ function cleanup() {
 }
 
 describe('c-quantic-generated-answer', () => {
-  beforeAll(() => {
-    mockSuccessfulHeadlessInitialization();
-  });
-
   afterEach(() => {
     cleanup();
   });
 
+  describe('when an initialization error occurs', () => {
+    beforeEach(() => {
+      mockErroneousHeadlessInitialization();
+    });
+
+    it('should display the initialization error component', async () => {
+      const element = createTestComponent();
+      await flushPromises();
+
+      const initializationError = element.shadowRoot.querySelector(
+        selectors.initializationError
+      );
+
+      expect(initializationError).not.toBeNull();
+    });
+  });
+
+  describe('when an RGA retryable error occurs', () => {
```

Comment:
I think `an RGA` is the correct way 🤨

Reply:
wow wtf you're right I just learned that today for acronyms it's what the letter "sounds" like? lol what a bogus rule...

because it's "arr-gee-ay" we should use "an" :joy: